spec/web_helpers.rb

```diff
@@ -0,0 +1,17 @@
+def sign_up(name: 'Ivan Sathianathan', username: 'ivan', email: 'ivan@test.com', password: '12345', password_confirmation: '12345')
```

Comment:
Metrics/LineLength: Line is too long. [131/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)